iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -86,20 +86,20 @@ constexpr uint32_t MAX_GATEWAY_SERVICES = 2 * MAX_CHANNEL_NUMBER;
 // Client
 constexpr uint32_t MAX_CLIENTS = build::IOX_MAX_SUBSCRIBERS;
 constexpr uint32_t MAX_REQUESTS_ALLOCATED_SIMULTANEOUSLY = 4U;
-constexpr uint32_t MAX_RESPONSES_PROCESSED_SIMULTANEOUSLY = 16U;
-constexpr uint32_t MAX_RESPONSE_QUEUE_CAPACITY = 16U;
+constexpr uint32_t MAX_RESPONSES_PROCESSED_SIMULTANEOUSLY = build::IOX_MAX_RESPONSES_PROCESSED_SIMULTANEOUSLY;
```

Comment:
Ah yes, I can answer my own question now. We had the options but they were  hard-coded here.

However, it is somewhat inconsistent as some of them are still hard-coded. No need for changes, just an observation. We do not want some of them configurable, but the distinction may look arbitrary and not cleanly separated. Something for another issue, maybe.

Reply:
Absolutely, I was also unsure about this inconsistency [here](https://github.com/eclipse-iceoryx/iceoryx/pull/1804#issuecomment-1327280593) . We can easily add at least the obvious ones, such as `MAX_REQUESTS_PROCESSED_SIMULTANEOUSLY`. I would make sense to me, I guess.